src/text.rs

```diff
@@ -0,0 +1,210 @@
+use std::path::PathBuf;
+use std::rc::Rc;
+use std::cell::RefCell;
+use std::fmt;
+
+use gfx_glyph;
+use gfx_glyph::{GlyphBrush, GlyphBrushBuilder, OwnedSection, Scale, StandardLineBreaker};
```

Comment:
any reason for some of the types being imported but not the others in `gfx_glyph`?
I find it easier to just shorthand the alias, aka `use gfx_glyph as g`.

Reply:
Makes sense, yes.